syserol/figures/figure10.py

```diff
@@ -0,0 +1,77 @@
+"""
+This creates Figure 3 for the Paper.
+"""
+
+from syserol.tensor import perform_CMTF
+from syserol.dataImport import createCube, getAxes, load_file, importGlycan
+from .common import subplotLabel, getSetup
+import pandas as pd
+import numpy as np
+import seaborn as sns
+
+def makeFigure():
+    """ Generate Figure 3 for Paper, Showing Better Interpretation of All Data from Decomposed Tensor"""
+    ax, f = getSetup((25, 25), (3, 4))
```

Comment:
The figure size is in units, and it should follow what it would actually be sized. For example, width no larger than 8, height no more than ~8.

Reply:
For now, are we leaving it as 25x25? The reason I made it so large is because when it was smaller, the figure bunched together in a way that was very difficult to read, and I got the error message: "UserWarning: constrained_layout not applied.  At least one axes collapsed to zero width or height.
  fig.canvas.print_figure(bytes_io, **kw)"

Reply:
It should follow the actual size, even if it bunches up. The bunching issue should be fixed with other changes (we can make later).